testing/tests/DevExpress.viz.charts/zoomAndPan.tests.js

```diff
@@ -2583,6 +2583,56 @@ QUnit.test("On pan", function(assert) {
     assert.equal(this.trackerStopHandling.callCount, 3);
 });
 
+QUnit.test("Two value axis. Pan", function(assert) {
```

Comment:
Please, rename this test so its name consists of action, condition, and the expected result
For example, "Pan action in pane without zoom if another pane has a zoom - default event behavior is not prevented"